packages/lesswrong/components/ea-forum/EAHomeCommunityPosts.tsx

```diff
@@ -27,7 +27,7 @@ const EAHomeCommunityPosts = ({classes}: {classes: ClassesType<typeof styles>})
     section: "community",
     onExpandEvent: "communityPostsSectionExpanded",
     onCollapseEvent: "communityPostsSectionCollapsed",
-    defaultExpanded: "loggedIn",
+    defaultExpanded: "all",
```

Comment:
@oetherington Sorry I'm adding more bits to this PR, but I wanted to make this change and it seemed ok to throw it in here 😄